dts/bindings/riscv/openisa,rv32m1-pcc.yaml

```
@@ -1,7 +1,7 @@
 # Copyright (c) 2018 Foundries.io
 # SPDX-License-Identifier: Apache-2.0
 
-description: RV32M1 PCC (Peripheral Clock Control) IP node
+description: RV32M1 PCC (Peripheral Clock Control) IP
```

"IP" can also go away afaict

```suggestion
description: RV32M1 PCC (Peripheral Clock Control)
```